Documentation/README_ManipulationHandler.md

```diff
@@ -78,6 +78,15 @@ Specifies on which axis the object will rotate when interacted with.
 **Constraints on Movement**
 * *None*
 * *Fix distance from head*
+* *Translation Axes* - If this option is selected, movement will be constrained to the axes specified in "Translation Axes"
+
+**Translation Axes**
+This will only be used if "Constraints on Movement" is set to "Translation Axes." These axes indicate
```

Comment:
If we are only setting this  value when constraints on movement is set, can we have a custom inspector that only shows these properties when that checkbox is checked? Niall has some great examples of how this has helped to simplify the new ObjectManipulator in his PR.